packages/web/src/components/logos.tsx

```diff
@@ -5,103 +5,36 @@ type PropsType = {
   className?: string;
 };
 
-/*
- * Our logo comes in different widths based on the size.
- */
 export function SrcbookLogo(props: PropsType) {
```

Comment:
This is a regression. There are 3 different logos based on the size of the square, otherwise the ratio of the vertices and the nodes becomes off. We need to reintroduce.